src/V8JsiRuntime.cpp

```diff
@@ -740,7 +740,7 @@ jsi::Value V8Runtime::ExecuteString(const v8::Local<v8::String> &source, const s
         v8::ScriptCompiler::CachedData *codeCache = v8::ScriptCompiler::CreateCodeCache(script->GetUnboundScript());
 
         jsi::ScriptSignature scriptSignature = {sourceURL, 1};
-        jsi::JSRuntimeSignature runtimeSignature = {"V8", 76};
+        jsi::JSRuntimeSignature runtimeSignature = {"V8", V8_MAJOR_VERSION * 10000 + V8_MINOR_VERSION * 1000 + V8_BUILD_NUMBER};

```

Comment:
>V8_MAJOR_VERSION * 10000 [](http://example.com/codeflow?start=58&length=24)

(nit) can you create a function returning this signature to avoid duplicated calculation